.github/workflows/system-test.yml

```diff
@@ -19,6 +19,8 @@ jobs:
       pull-requests: read
     steps:
       - uses: elastic/apm-pipeline-library/.github/actions/is-pr-approved@current
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this something like a predefined secret? I cannot find it in https://github.com/elastic/apm-queue/settings/secrets/actions